docs/src/conf.py

```diff
@@ -96,7 +96,7 @@ def setup(app):
     "ase": ("https://wiki.fysik.dtu.dk/ase/", None),
     "python": ("https://docs.python.org/3", None),
     "torch": ("https://pytorch.org/docs/stable/", None),
-    "metatensor": ("https://lab-cosmo.github.io/metatensor/latest/", None),
+    "metatensor": ("https://metatensor.github.io/metatensor/latest/", None),
```

Comment:
```suggestion
    "metatensor": ("https://docs.metatensor.org/latest/", None),
```

Let's not rely on the github pages URL which we might stop using at some point.